install_config/configuring_azure.adoc

```diff
@@ -11,26 +11,49 @@
 toc::[]
 
 == Overview
-{product-title} can be configured to access an
-link:https://azure.microsoft.com/en-us/services/storage/disks/[Azure
+{product-title} can be configured to access
+link:https://azure.microsoft.com/en-us/services/storage/disks/[Microsoft Azure
 infrastructure], including
 xref:../install_config/persistent_storage/persistent_storage_azure.adoc#install-config-persistent-storage-persistent-storage-azure[using
-Azure disk as persistent storage] for application data. After Azure is
+Azure disk as persistent storage] for application data. After Microsoft Azure is
 configured properly, some additional configurations need to be completed on the
 {product-title} hosts.
 
 [[configuring-azure-permissions]]
 == Permissions
-Configuring Azure for {product-title} requires the following role:
+Configuring Microsoft Azure for {product-title} requires the following role:
 
 [cols='1,3']
 |===
 
 | Contributor
-| To create and manage all types of Azure resources.
+| To create and manage all types of Microsoft Azure resources.
 
 |===
 
+[[configuring-azure-prerequisites]]
+== Prerequisites
+
+* If you are using Azure Disk as an OpenShift Persistent Volume for  {product-title} version 3.7 or lower, you must enable Azure Cloud Provider.
++
+[NOTE]
+====
+This is not required if you are using Azure File, Azure Blob storage for docker
+registry, or Azure Active Directory OpenID Connect for master authentication.
+====
+
+* All virtual machines (VM) configured as OpenShift nodes, must belong to a single
+resource group.
+* Microsoft Azure VM names:
+  ** Must be same as OpenShift node names, and
+  ** These names must not be included any capital letters.
+* If you plan to use Azure Managed Disks:
+  ** {product-title} version 3.7 or later is required.
+  ** You must create VMs with Azure Managed Disks.
+* If you plan to use unmanaged disks:
+  ** You must create VMs with unmanaged disks.
+* Please be aware of node names. All nodes in OpenShift cluster have to communicate with their names. It means you have to configure DNS to resolve private IP from node names. If you create Azure VNet/VNIC/VM by default, you don't have to configure anything about DNS because Azure default DNS can do that. If you use custom DNS for OpenShift cluster, you have to configure by yourself.
```

Comment:
@tanaka-takayoshi 

I am not entirely sure what this means, can you help me re-write this?
```
Please be aware of node names. All nodes in OpenShift cluster have to communicate with their names. It means you have to configure DNS to resolve private IP from node names. If you create Azure VNet/VNIC/VM by default, you don't have to configure anything about DNS because Azure default DNS can do that. If you use custom DNS for OpenShift cluster, you have to configure by yourself.
```